frontend/src/utils/auth.ts

```diff
@@ -0,0 +1,38 @@
+import firebase from 'firebase';
+import { config } from 'dotenv';
+import { AxiosRequestConfig } from 'axios';
+
+config();
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyCJulo-7tVPqEEsTTRjEsUzSRw8-RCLDVw',
```

Comment:
Should we really be exposing our API key like this? Even if it's for firebase auth, maybe it's best to make a service account file and generate a new API key

Reply:
[Samwise](https://github.com/cornell-dti/samwise/blob/master/frontend/src/firebase/index.ts) as well as many other projects do it so I assume it's safe